### PR TITLE
Added support for mods without vals

### LINKED ISSUE #116: merge bemdecl: lost modifier without values
`bemdecl-1.js`
```js
exports.blocks = [{
    name: 'block-1',
    mods: [{ name: 'mod' }]
}];
```

`bemdecl-2.js`
```js
exports.blocks = [
    { name: 'block-2' }
];
```

If merge `bemdecl-1.js` with `bemdecl-2.js`:

```js
exports.blocks = [
    { name: 'block-1', },
    { name: 'block-2' }
];
```

expected:
```js
exports.blocks = [
    {
        name: 'block-1',
        mods: [{ name: 'mod' }]
    },
    { name: 'block-2' }
];
```

### PR DESCRIPTION
**Pull request by blond**
Resolved #116

### CODE REVIEW

File: lib/deps/deps.js [hunk 1]
```diff
@@ -59,6 +59,8 @@ module.exports = {
                             mod.vals.forEach(function (val) {
                                 res.push({ block: block.name, mod: mod.name, val: val.name });
                             });
+                        } else {
+                            res.push({ block: block.name, mod: mod.name });
```
Comment (zxqfox) [inline lib/deps/deps.js]: Btw, bem-naming can't do this work for ya?

Comment (blond) [inline lib/deps/deps.js]: What can help `bem-naming`?

Comment (zxqfox) [inline lib/deps/deps.js]: To make `{ block, elem }` or `{block, mod}`, or `{block, elem, mod}` objects.

**Comment by tadatuta** [comment]
lgtm

File: lib/deps/deps.js (hunk 1 above)
Comment (blond) [inline lib/deps/deps.js]: There is a transformation of the BEMDECL v1.0 to inner ENB format. `bem-naming` should not be aware of declarations and their formats. To solve this task may be suitable `bem-decl`.

**Pull request merged**